.github/workflows/create-release.yml

```diff
@@ -46,7 +51,7 @@ jobs:
         with:
           repo_token: ${{ secrets.GITHUB_TOKEN }}
           tag: ${{ github.ref }}
-          file: v2/bin/azureserviceoperator_*.yaml
+          file: "v2/bin/{azureserviceoperator,multitenant-cluster,multitenant-tenant}_*.yaml"
```

Comment:
I got a bit worried about whether the braces would be respected in the pattern but it [seems like they should be](https://www.npmjs.com/package/glob). 
(I haven't tested the release action.)